apps/web/src/routes/_content/_misc/professors/$category/index.tsx

```diff
@@ -67,6 +67,10 @@ export function ProfessorCategoryPage() {
     }
   });
 
+  const sortedProfessors = filteredProfessors?.sort((a, b) =>
```

Comment:
It's better to create a shallow copy ([...filteredProfessors]) to avoid changes on original array